imagepipeline/src/main/java/com/facebook/imagepipeline/core/ImagePipelineConfig.java

```diff
@@ -80,6 +80,7 @@
   private final DiskCacheConfig mMainDiskCacheConfig;
   private final MemoryTrimmableRegistry mMemoryTrimmableRegistry;
   private final NetworkFetcher mNetworkFetcher;
+  private int mHttpNetworkTimeout = -1;
```

Comment:
can you make this final please?

Reply:
Marked as final and modified code in ImagePipelineConfig to handle change to final declaration.